### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by anupkgurung**
added login + signup page
added feature to login and singup

loginSM
singupSM

### CODE REVIEW

**Comment by varun-Tad** [comment]
looks good. You can go ahead and merge.

**Comment by anupkgurung** [comment]
> looks good. You can go ahead and merge.

you need to approve the pull request

File: src/features/Authentication/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+export const Login = () => {
+
+    return (
+        <div className="flex justify-center pt-20">
+            <div className="p-6 rounded-lg shadow-lg bg-white max-w-sm w-96">
+                <form>
+                    <div className="form-group mb-6">
+                        <label htmlFor="email" className="form-label flex mb-2 text-gray-700">Email address</label>
+                        <Input type={"email"}
```
Comment (aiman-ansari) [inline src/features/Authentication/Login.jsx:13]: I think there is no need to put email in {}

File: src/features/Authentication/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        lastName : "zona",
+        username:"abc@gmail.com",
+        password : '1234'
+    }
+
+    const signup = (e) => {
+       
+        e.preventDefault()
+        dispatch(userSignup(initialSingupData))
+        //Navigate(location)
```
Comment (aiman-ansari) [inline src/features/Authentication/Signup.jsx:25]: Suggestion: Remove console log

**Review by aiman-ansari — APPROVED** [review]
Code is easy to understand. Variable naming is also declarative .
Some suggestions are given do check .

**Review by rohanmathur91 — COMMENTED** [review]
- Clean code
- Good naming convention
- Looks good to merge

**Pull request merged**